h2/src/test/org/h2/test/store/TestMVStore.java

```diff
@@ -1416,7 +1416,9 @@ private void testFastDelete() {
             assertEquals(i + 1, m.size());
         }
         assertEquals(1000, m.size());
-        assertEquals(131896, s.getUnsavedMemory());
+        // previously (131896) we fail to account for initial root page for every map
+        // there are two of them here (meta and "data"), hence lack of 256 bytes
+        assertEquals(132152, s.getUnsavedMemory());
```

Comment:
It was not an intentional fix, but rather a byproduct. Plus this "exact" memory accounting is more like [counting angels dancing on the pin](https://en.wikipedia.org/wiki/How_many_angels_can_dance_on_the_head_of_a_pin%3F). Result is used only once - compared against arbitrary configuration parameter in order to make an optional decision about extra commit. So IMHO,  who cares? 8-)